inflate_p.h

```diff
@@ -5,6 +5,8 @@
 #ifndef INFLATE_P_H
 #define INFLATE_P_H
 
+#include "functable.h"
```

Comment:
What's the include here needed for?

Reply:
It's isn't, I had been messing with dispatching to chunkmemset_safe et al but didn't see a measurable improvement.